pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools>=44", "wheel"]
```

Comment:
This seems unrelated. We follow the process [here](https://pip.pypa.io/en/stable/development/release-process/#creating-a-new-release) when making a new release. Since that would pick up the latest setuptools, no need to update pyproject.toml.